### PR TITLE
chore(bigcommerce): add documentation related to cache webhooks (FC-1032)

### PR DESCRIPTION
**Pull request by paul-vd**
[Overview Page](https://deploy-preview-429--heuristic-almeida-1a1f35.netlify.app/docs/bigcommerce/overview.html)
[Webhooks Page](https://deploy-preview-429--heuristic-almeida-1a1f35.netlify.app/docs/bigcommerce/webhooks.html)

### CODE REVIEW

**Comment by paul-vd** [comment]
FC-1032

**Review by Kagaminara — COMMENTED** [review]
few nits

File: source/docs/bigcommerce/webhooks.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+description: Create webhooks to recieve events from BigCommerce for cache invalidation and other purposes.
+---
+
+## Setting up Front-Commerce
+
+First you will need to create a webhook secret within your Front-Commerce app, to do this, you can add the `FRONT_COMMERCE_BIG_COMMERCE_WEBHOOK_SECRET` environment variable to your `.env` file.
+
+> **ProTip:** create a random secret by running `openssl rand -hex 25` in your terminal.
+
+Thats all, you should now be ready to recieve webhooks on your application using your webhook secret.
```
[the remark this answers is not in the document]
Comment (paul-vd) [reply source/docs/bigcommerce/webhooks.md:13]: lmao... there were other points when I wrote that line 🙃 hehehe

File: source/docs/bigcommerce/webhooks.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+id: bigcommerce-webhooks
+title: Webhooks
+description: Create webhooks to recieve events from BigCommerce for cache invalidation and other purposes.
+---
+
+## Setting up Front-Commerce
+
+First you will need to create a webhook secret within your Front-Commerce app, to do this, you can add the `FRONT_COMMERCE_BIG_COMMERCE_WEBHOOK_SECRET` environment variable to your `.env` file.
```
Comment (abailleulfc) [inline source/docs/bigcommerce/webhooks.md:9]:
```suggestion
You need to setup the webhook secret within your Front-Commerce app with the `FRONT_COMMERCE_BIG_COMMERCE_WEBHOOK_SECRET` environment variable.
```

Team "shorter is better" :sweat_smile:

File: source/docs/bigcommerce/webhooks.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- `products/*`
+- `sku/*`
+
+To do this we first need to setup Authentication for your BigCommerce account, you can refer to the [documentation](https://developer.bigcommerce.com/docs/ZG9jOjIyMDYxMw-v2-and-v3-rest-api-authentication).
+
+We will need to create the webhook api to get an Access Token and the Store Hash
+
+1. Navigate to your BigCommerce account and click on the `Settings` tab, scroll down and open `API > API accounts`.
+1. Click on the `+ Create API Account` button.
+1. Copy the `API path` as it contains the `Store Hash`. (see figure below)
```
Comment (abailleulfc) [inline source/docs/bigcommerce/webhooks.md:29]: Why not putting the store hash image just under this point ?

File: source/docs/bigcommerce/webhooks.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+## Setting up BigCommerce
+
+We will now create webhooks for the following scopes:
+
+- `catalog/*`
+- `products/*`
+- `sku/*`
+
+To do this we first need to setup Authentication for your BigCommerce account, you can refer to the [documentation](https://developer.bigcommerce.com/docs/ZG9jOjIyMDYxMw-v2-and-v3-rest-api-authentication).
```
Comment (abailleulfc) [inline source/docs/bigcommerce/webhooks.md:23]:
```suggestion
We first need to setup Authentication for your BigCommerce account (you can refer to the [documentation](https://developer.bigcommerce.com/docs/ZG9jOjIyMDYxMw-v2-and-v3-rest-api-authentication)).
```
nitpick : just seems more readable like this :innocent:

File: source/docs/bigcommerce/webhooks.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  <img src="./assets/new-webhook-storehash.png" alt="Store Hash in BigCommerce API path" style="border-radius:5px;">
+</div>
+
+#### Access Token
+
+<div style="text-align:left;">
+  <img src="./assets/new-webhook-accesstoken.png" alt="SBigCommerce API account access token" style="border-radius:5px;">
+</div>
+
+You are now ready to get started with your webhooks, we have created a [BigCommerce Webhook API](https://documenter.getpostman.com/view/16678499/UzQys4QZ) in postman to simplify this process, you can hit the `Run in Postman` button and simply replace all the relevant varialbles to get started [creating your webhooks](#Creating-a-webhook).
```
Comment (abailleulfc) [inline source/docs/bigcommerce/webhooks.md:55]:
```suggestion
You are now ready to get started with your webhooks, we have created a [BigCommerce Webhook API](https://documenter.getpostman.com/view/16678499/UzQys4QZ) in postman to simplify this process, you can hit the `Run in Postman` button and simply replace all the relevant variables to get started [creating your webhooks](#Creating-a-webhook).
```

File: source/docs/bigcommerce/webhooks.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+### Creating a webhook
+
+You can refer to the Create folder in the [BigCommerce Webhook API](https://documenter.getpostman.com/view/16678499/UzQys4QZ), to create the correct webhooks.
+
+> **Important:** All those webhooks are required for cache invalidation.
+
+### Updating a webhook
+
+To update a webhook, you first need to run the `GET ALL WEBHOOKS` api, it will return an array of webhooks, you can then replace the `WEBHOOK_ID` variable with the id of the webhook that you want to remove.
```
Comment (abailleulfc) [inline source/docs/bigcommerce/webhooks.md:65]:
```suggestion
To update a webhook, you first need to run the `GET ALL WEBHOOKS` api to get an array of webhooks. You can then replace the `WEBHOOK_ID` variable with the id of the webhook that you want to remove.
```

**Review by abailleulfc — CHANGES_REQUESTED** [review]

**Review by Kagaminara — APPROVED** [review]

File: source/docs/bigcommerce/webhooks.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+description: Create webhooks to receive events from BigCommerce for cache invalidation and other purposes.
+---
+
+## Setting up Front-Commerce
+
+You need to setup the webhook secret within your Front-Commerce app with the `FRONT_COMMERCE_BIG_COMMERCE_WEBHOOK_SECRET` environment variable.
+
+> **ProTip:** create a random secret by running `openssl rand -hex 25` in your terminal.
+
+Thats all, you should now be ready to recieve webhooks on your application using your webhook secret.
```
Comment (real34) [inline source/docs/bigcommerce/webhooks.md:13]:
```suggestion
Thats all, you should now be ready to receive webhooks on your application using your webhook secret.
```

**Review by real34 — APPROVED** [review]

**Pull request merged**